release-notes/source/linux-default.inc

```diff
@@ -79,6 +80,6 @@ t2080rdb                linux-qoriq          5.4.3+git
 t2080rdb-64b            linux-qoriq          5.4.3+git
 t4240rdb                linux-qoriq          5.4.3+git
 t4240rdb-64b            linux-qoriq          5.4.3+git
-ventana                 linux-fslc           5.4.58+git
```

Comment:
Let me send the PR to try to fix this on meta-freescale, and them I update this PR

Reply:
https://github.com/Freescale/meta-freescale-3rdparty/pull/105